src/blackboxTest/bash/TestMultiThreading.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+TESTPATH=$1
+EXIT_CODE=0
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+WHITE='\033[0m'
+
+
+runThreads() {
+  ./requestWithDelay.sh ${TESTPATH} > /dev/null 2>/dev/null &
```

Comment:
Could _probably_ cleanup a little duplication here. The two files are almost identical except that they call two different files that are also almost identical. The only thing that differs there is the output file to which they write and the duration which they sleep.

Why the redirect?